### DISCUSSION TITLE
Keyboard Detection Problems

### DISCUSSION THREAD

**Discussion by Sebasli18** [ppy/osu#26447] [Q&A]
### Type

Game behaviour

### Bug description

I have a problem with keyboard detection. It happens that when I play Osu! stable the keyboard detects it without any problem. But when I play Osu! lazer detects in certain areas such as in the configuration search engine or the map search engine it does detect it, but within gameplay, configuring keys, or other moments it does not detect it. In addition to this, it does not detect the escape and delete key at all times. It should be noted that I use a laptop keyboard and a mechanical one connected to it and neither of them detect it.

### Version

2023.1231.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13855552/compressed-logs.zip)

**Comment by goury212** [comment]
I really appreciate your help!

If you want to test the keyboard keys, if your keyboard is not working well, if there are [faulty keyboards](https://keyboardkeytest.com/blogs/how-to-keep-your-keyboard-in-good-shape), or if you want to try your computer keyboard, then this free online tool is for you.
